backend/routes/announcements.js

```diff
@@ -1,49 +1,60 @@
-const express = require('express');
+const express = require("express");
 const router = express.Router();
-const { Announcements} = require('../database');
+const { Announcements } = require("../database");
 
 // Routes
-
+//todo:create an update for announcements
 router.use((request, response, next) => {
   response.header("Access-Control-Allow-Origin", "*");
-  response.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
+  response.header(
```

Comment:
In a PM can you explain to me how this actually works? Does it just allow the front end and backend to communicate since they're not on the same server ?